src/VocabGameData.js

```diff
@@ -0,0 +1,39 @@
+const basePairs = [
+  ["hello", "bonjour"],
+  ["goodbye", "au revoir"],
+  ["please", "s'il vous plaît"],
+  ["thank you", "merci"],
+  ["thank you very much", "merci beaucoup"],
+  ["you're welcome", "de rien"],
+  ["excuse me", "excusez-moi"],
+  ["yes", "oui"],
+  ["no", "non"],
+  ["can you help me, please?", "pouvez-vous m’aider, s’il vous plaît?"]
+];
+
+let dataset = [];
+function createDataset(base) {
+  base.forEach(function(array) {
+    dataset.push([
+      array[0],
+      {
+        word: array[0],
+        translation: array[1],
+        isSelected: null,
+        isCorrect: null
+      }
+    ],
+    [
+      array[1],
+      {
+        word: array[1],
+        translation: array[0],
+        isSelected: null,
+        isCorrect: null
+      }
+    ]);
+  });
+}
+
+createDataset(basePairs);
+let datasetMap = new Map(dataset);
```

Comment:
ok, I would tuck in the new Map() conversion within the function `createDataset`

so then all we have  to do is run the function, and it spits out the Map dataset without doing another step.

Reply:
let me know when u made the change and ill pull!

Reply:
@serpient I made the changes!